route53_crossfeed_app.tf

```diff
@@ -308,6 +308,18 @@ resource "aws_route53_record" "crossfeed_staging_acme_TXT" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "integration_crossfeed_staging_acme_TXT" {
```

Comment:
It's not clear to me if "integration" is considered part of "Staging", but if not, it should be given a separate section.  I also think the resource name format should match the others.
```suggestion
# ------------------------------------------------------------------------------
# Integration entries
# ------------------------------------------------------------------------------

resource "aws_route53_record" "crossfeed_integration_acme_TXT" {
```